### PR TITLE
Add configuration for the GitHub issue template chooser

### PR DESCRIPTION
**Pull request by erimatnor**
Configure the GitHub issue template chooser to include options with
links to the page about support requests, the Slack community, and
documentation changes.

### CODE REVIEW

**Review by mkindahl — APPROVED** [review]

**Review by fabriziomello — APPROVED** [review]

**Comment by erimatnor** [comment]
I will merge this after all other templates have been merged since this configuration disables the ability to file blank issues.

**Review by svenklemm — APPROVED** [review]

**Pull request merged**
